apps/test/unit/instructionsComponentsTest.js

```diff
@@ -10,73 +10,67 @@ describe('instructions components', () => {
+      const element = wrapper.find('.instructions-markdown').first();
       expect(element.props().style.paddingTop).to.equal(19);
       expect(element.props().style.marginBottom).to.equal(35);
       expect(element.props().style.marginLeft).to.equal(undefined);
       expect(element.text()).to.equal('md');
     });
 
     it('inTopPane has no top padding', function() {
-      var dom = mount(
-        <div>
-          <StatelessMarkdownInstructions
-            markdown="md"
-            inTopPane={true}
-            noInstructionsWhenCollapsed={true}
-          />
-        </div>
+      const wrapper = mount(
+        <StatelessMarkdownInstructions
+          markdown="md"
+          inTopPane={true}
+          noInstructionsWhenCollapsed={true}
+        />
       );
-      var element = dom.find('.instructions-markdown').first();
+      const element = wrapper.find('.instructions-markdown').first();
       expect(element.props().style.paddingTop).to.equal(0);
     });
   });
 
   describe('NonMarkdownInstructions', function() {
     it('can have just instructions', function() {
-      var dom = mount(
-        <div>
-          <NonMarkdownInstructions
-            puzzleTitle="title"
-            shortInstructions="instructions"
-          />
-        </div>
+      const wrapper = mount(
+        <NonMarkdownInstructions
+          puzzleTitle="title"
+          shortInstructions="instructions"
+        />
       );
-      var elements = dom.childAt(0).children();
+      const elements = wrapper
+        .find('div')
+        .first()
+        .children();
       expect(elements.length).to.equal(2);
-      expect(elements.first())
-        .text()
-        .to.equal('title');
-      expect(elements.last())
-        .text()
-        .to.equal('instructions');
+      expect(elements.first().text()).to.equal('title');
+      expect(elements.last().text()).to.equal('instructions');
     });
 
     it('can have both instructions and instructions2', function() {
-      var dom = mount(
-        <div>
-          <NonMarkdownInstructions
-            puzzleTitle="title"
-            shortInstructions="instructions"
-            instructions2="instructions2"
-          />
-        </div>
+      const wrapper = mount(
+        <NonMarkdownInstructions
+          puzzleTitle="title"
+          shortInstructions="instructions"
+          instructions2="instructions2"
+        />
       );
-      var elements = dom.childAt(0).children();
+      const elements = wrapper
+        .find('div')
+        .first()
+        .children();
       expect(elements.length).to.equal(3);
-      expect(elements.get(0).textContent).to.equal('title');
-      expect(elements.get(1).textContent).to.equal('instructions');
-      expect(elements.get(2).textContent).to.equal('instructions2');
+      expect(elements.at(0).text()).to.equal('title');
+      expect(elements.at(1).text()).to.equal('instructions');
+      expect(elements.at(2).text()).to.equal('instructions2');
```

Comment:
Because of the change to the definition of "children", `get` (which can return a react node or a DOM node) didn't necessarily give us a node with `textContent`.  Instead I'm using `at` which returns a `ReactWrapper` so we can use the `text()` helper that walks the tree to give us the same `textContent` that we wanted to test.  (This is also a better reflection of our approach in the test above, as `first` and `last` also return a `ReactWrapper`.)